src/esp/assets/ResourceManager.h

```diff
@@ -256,8 +276,8 @@ class ResourceManager {
    * initialization that may be required when @ref metadataMediator_ is changed.
    * @param _MM a reference to the new @ref metadataMediator_.
    */
-  void setMetadataMediator(metadata::MetadataMediator::ptr _MM) {
-    metadataMediator_ = _MM;
+  void setMetadataMediator(metadata::MetadataMediator::ptr MM) {
+    metadataMediator_ = std::move(MM);
```

Comment:
Merge error?

Reply:
Nah, this is right. Clang-Tidy recommends this for performance and style.